ios/FluentUI.Demo/FluentUI.Demo/Demos/ButtonDemoController_SwiftUI.swift

```diff
@@ -0,0 +1,110 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import FluentUI
+import SwiftUI
+import UIKit
+
+class ButtonDemoControllerSwiftUI: UIHostingController<ButtonDemoView> {
+    override init?(coder aDecoder: NSCoder, rootView: ButtonDemoView) {
+        preconditionFailure("init(coder:) has not been implemented")
+    }
+
+    @objc required dynamic init?(coder aDecoder: NSCoder) {
+        preconditionFailure("init(coder:) has not been implemented")
+    }
+
+    init() {
+        super.init(rootView: ButtonDemoView())
+        self.title = "Button Vnext (SwiftUI)"
+    }
+}
+
+struct ButtonDemoView: View {
+    @State var isDisabled: Bool = false
+    @State var text: String = "Button"
+    @State var showImage: Bool = true
+    @State var showLabel: Bool = true
+    @State var showAlert: Bool = false
+    @State var size: MSFButtonSize = .large
+    @State var style: MSFButtonStyle = .accentFloating
+
+    public var body: some View {
+        VStack {
+            MSFButtonView(action: {
```

Comment:
I noticed this button isn't responding to theme color changes (it's still blue when the main vNext view is green). In fact, that seems to be a problem for all of our SwiftUI demos...

Reply:
Yes. This is a known issue (likely with the timing that the HostingController is being added to the window). I have created a bug in ADO to track it.